libs/examples/src/quick-filter/BUILD.bazel

```diff
@@ -1,13 +1,22 @@
 load("//tools/bazel_rules:index.bzl", "ng_module")
 
-package(default_visibility = ["//:__subpackages__"])
+package(default_visibility = ["//visibility:public"])
 
 ng_module(
-    name = "quick-filter",
-    srcs = glob(["**/*.ts"]),
-    angular_assets = glob(["**/*.html"]),
-    module_name = "@dynatrace/examples/quick-filter",
-    tsconfig = "//libs/examples:tsconfig-lib",
+    name = "compile",
```

Comment:
This name is referenced in the e2e test app, we might need to update it there.

https://github.com/dynatrace-oss/barista/blob/a59f8815b59c71b9e95f6a9e14d8b3ac4f762c49/apps/components-e2e/src/components/quick-filter/BUILD.bazel#L23